src/components/Resource/ResourceCreate.tsx

```diff
@@ -300,8 +300,11 @@ export default function ResourceCreate(props: resourceProps) {
         <TextFormField
           label={t("required_quantity")}
           name="requested_quantity"
-          value={state.form.required_quantity}
+          type="number"
+          min={1}
+          value={state.form.requested_quantity}
           onChange={handleChange}
+          onInput={sanitizeNumberInput}
```

Comment:
Let's not change what the user enters, trigger the error message in validateForm.